resources/datadog-agent/msi/source.wxs.erb

```diff
@@ -223,6 +233,12 @@
       <ComponentRef Id="RegisterConfVariables" />
     </Feature>
 
+    <InstallExecuteSequence>
+      <Custom Action="WixCloseApplications"
+          After="StopServices"><![CDATA[REMOVE="ALL" OR REINSTALL]]></Custom>
```

Comment:
Yes, I found this explanation about the syntax - https://www.firegiant.com/wix/tutorial/com-expression-syntax-miscellanea/expression-syntax/, wondering about the variables